src/game/uo_files/CUOMobtypes.h

```diff
@@ -0,0 +1,62 @@
+/**
+* @file CUOMobtypes.h
+*
+*/
+
+#ifndef _INC_CUOMOBTYPES_H
+#define _INC_CUOMOBTYPES_H
+
+#include <vector>
+#include "../../common/common.h"
+
+// All these structures must be byte packed.
+#if defined(_WIN32) && defined(_MSC_VER)
+    // Microsoft dependant pragma
+#pragma pack(1)
+#define PACK_NEEDED
+#else
+    // GCC based compiler you can add:
+#define PACK_NEEDED __attribute__ ((packed))
+#endif
+
+/**
+* mobtypes.txt
+*/
+struct CUOMobTypesType
```

Comment:
You could simplify this as we don't need structure packing here. It's usually needed when we want to read a sequence of bytes (from a UO mul file) and with those fill an entire struct. Structure packing instructs the compiler to not add padding bytes.
Pro read for more detail: http://www.catb.org/esr/structure-packing/
Faster answer: https://stackoverflow.com/questions/4306186/structure-padding-and-packing

Reply:
Exactly